modules/telco-ran-ptp-operator.adoc

```diff
@@ -1,40 +1,26 @@
 // Module included in the following assemblies:
 //
-// * scalability_and_performance/telco_ref_design_specs/ran/telco-ran-ref-du-components.adoc
+// * scalability_and_performance/telco_ran_du_ref_design_specs/telco-ran-du-rds.adoc
 
 :_mod-docs-content-type: REFERENCE
 [id="telco-ran-ptp-operator_{context}"]
 = PTP Operator
 
 New in this release::
-* A new version two of the Precision Time Protocol (PTP) fast event REST API is available.
-Consumer applications can now subscribe directly to the events REST API in the PTP events producer sidecar.
-The PTP fast event REST API v2 is compliant with the link:https://orandownloadsweb.azurewebsites.net/download?id=344[O-RAN O-Cloud Notification API Specification for Event Consumers 3.0].
-You can change the API version by setting the `ptpEventConfig.apiVersion` field in the `PtpOperatorConfig` resource.
+* No reference design updates in this release
 
 Description::
-See "Recommended {sno} cluster configuration for vDU application workloads" for details of support and configuration of PTP in cluster nodes.
-The DU node can run in the following modes:
-+
-* As an ordinary clock (OC) synced to a grandmaster clock or boundary clock (T-BC).
-
-* As a grandmaster clock (T-GM) synced from GPS with support for single or dual card E810 NICs.
-
-* As dual boundary clocks (one per NIC) with support for E810 NICs.
-
-* As a T-BC with a highly available (HA) system clock when there are multiple time sources on different NICs.
-
-* Optional: as a boundary clock for radio units (RUs).
+Configure PTP in cluster nodes with `PTPConfig` CRs for the RAN DU use case with features like Grandmaster clock (T-GM) support via GPS, ordinary clock (OC), boundary clocks (T-BC), dual boundary clocks, High Availability (HA) and optional fast event notification over HTTP.
```

Comment:
```suggestion
Configure PTP in cluster nodes with `PTPConfig` CRs for the RAN DU use case with features like Grandmaster clock (T-GM) support via GPS, ordinary clock (OC), boundary clocks (T-BC), dual boundary clocks, High Availability (HA), and optional fast event notification over HTTP.
```
unless "High Availability (HA) and optional fast event notification over HTTP" are a single feature name.